src/components/mdSwitch/mdSwitch.vue

```diff
@@ -3,7 +3,7 @@
     <div class="md-switch-container" @click="toggleSwitch">
       <div class="md-switch-thumb" :style="styles" v-md-ink-ripple="disabled">
         <input type="checkbox" :name="name" :id="id" :disabled="disabled" :value="value">
-        <button class="md-switch-holder"></button>
+        <button :type="type" class="md-switch-holder"></button>
```

Comment:
If the user don't pass the type we must assume that this will have ```type="button"```
We need to change that to something like that:
``` html
<button :type="type || 'button'" class="md-switch-holder"></button>
```